src/cls/queue/cls_queue_ops.h

```diff
@@ -136,4 +136,26 @@ struct cls_queue_get_capacity_ret {
 };
 WRITE_CLASS_ENCODER(cls_queue_get_capacity_ret)
 
+struct cls_queue_get_entries_and_size_ret {
```

Comment:
maybe rename to `cls_queue_get_stats_ret`?
so, if we add more stats in the future, we don't need a new struct